slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -251,13 +251,18 @@ class SLAMServiceImpl final : public SLAMService::Service {
     // a pcd string with probability estimates written to the color field. The
     // pcd is generated from PaintedMapSlices() and sampled to fit the 32 MB
     // limit on gRPC messages. The sampled behavior may change when moving to
-    // streamed point clouds
+    // streamed point clouds.
     void GetLatestSampledPointCloudMapString(std::string &pointcloud);
 
     // BackupLatestMap extracts and saves the latest map as a backup in
     // the respective member variables.
     void BackupLatestMap();
 
+    // If using the LOCALIZING action mode, cache a copy of the map before
```

Comment:
[nit] can we either have the comment for this function be on the prototype or the function definition but not both? Otherwise, it makes it likely that the two comments will diverge over time, leading to confusion.

Reply:
meant to only have it on the prototype, removed it from the definition